.github/actions/delivery/action.yml

```diff
@@ -89,9 +89,15 @@ runs:
         done
       shell: bash
 
-    - if: ${{ inputs.distrib == 'bullseye' }}
+    - if: ${{ inputs.distrib == 'bullseye' || inputs.distrib == 'bookworm' }}
       name: Publish DEBs
       run: |
+        if [ ${{ inputs.distrib == 'bookworm' }} ]; then
+          if [[ ${{ inputs.version }} =~ 22.* ]] || [[ ${{ inputs.version }} =~ 23.* ]]; then
```

Comment:
the two conditions could be merged to reduce cyclomatic complexity

Reply:
you should skip the job directly in the workflow using github expression (`!contains('bookworm', matrix.distrib) || !startsWith('22.', version) || !startsWith('23.', version)`)